global.json

```diff
@@ -1,5 +1,5 @@
 {
   "sdk": {
-    "version": "3.0.100-preview-010184"
+    "version": "3.0.100-preview3-010313"
```

Comment:
Ingesting daily aspnetcore builds is going to be a pain. I need to figure out a better way to do this but for now, download from https://github.com/dotnet/core-sdk#installers-and-binaries. I need to find a way to get a permanent link to a specific version since https://dotnetcli.blob.core.windows.net/dotnet/Sdk/release/3.0.1xx/dotnet-sdk-latest-win-x64.zip will change frequently.

Reply:
@natemcmaster I don't have access to the dotnetcli blob store in the storage explorer so I can't find a permanent link. Do you have any other ideas on how to ingest daily builds more easily?

Reply:
Don't download from dotnetcli.blob.core.windows.net directly. This location is an implementation detail. I would recommend using `dotnet-install.sh`/`.ps1` instead. See https://docs.microsoft.com/dotnet/core/tools/dotnet-install-script for more details.

Example:
```ps1
$globalJson = Get-Content -Raw ./global.json | ConvertFrom-Json
Invoke-WebRequest https://dot.net/v1/dotnet-install.ps1 -out dotnet-install.ps1
./dotnet-install.ps1 -version $globalJson.sdk.version
```